src/routes/papers/index.js

```diff
@@ -30,9 +30,8 @@ function getBadgeCount(request, response) {
     });
 
   getBadges(function (error, badges) {
-    if (error !== null) {
-      console.log('Get error from return Badges ' + error);
-      response.send(error);
+    if (error !== null || !badges) {
+      response.json(0);
```

Comment:
hmm this one is interesting... I guess it would be the same if you are doing a check for number wherever you're using this, so `0` would be fine.